libs/SalesforceReact/package.json

```diff
@@ -6,21 +6,23 @@
         "start": "node node_modules/react-native/local-cli/cli.js start"
     },
     "dependencies": {
-        "chai": "4.1.2",
-        "react-native-force": "git+https://github.com/forcedotcom/SalesforceMobileSDK-ReactNative.git#dev",
-        "eslint": "^5.1.0",
-        "acorn": "^6.4.1",
-        "@react-native-community/cli": "^4.5.1",
-        "react": "16.11.0",
-        "react-native": "0.62.2"
+        "react": "16.13.1",
+        "react-native": "0.63.2",
+        "react-native-force": "git+https://github.com/wmathurin/SalesforceMobileSDK-ReactNative.git#rn63",
```

Comment:
Will switch to "git+https://github.com/forcedotcom/SalesforceMobileSDK-ReactNative.git#dev" before merging.